oci/layout/oci_delete_test.go

```diff
@@ -296,3 +295,61 @@ func assertBlobDoesNotExist(t *testing.T, blobsDir string, blobDigest string) {
 	_, err = os.Stat(blobPath)
 	require.True(t, os.IsNotExist(err))
 }
+
+// copyDir copies a whole directory src recursively to dst.
+func copyDir(src, dst string) error {
+	info, err := os.Stat(src)
+	if err != nil {
+		return err
+	}
+
+	if err := os.MkdirAll(dst, info.Mode()); err != nil {
+		return err
+	}
+
+	dirEntries, err := os.ReadDir(src)
+	if err != nil {
+		return err
+	}
+
+	for _, dirEntry := range dirEntries {
+		srcPath := filepath.Join(src, dirEntry.Name())
+		dstPath := filepath.Join(dst, dirEntry.Name())
+
+		copy := copyFile
+		if dirEntry.IsDir() {
+			copy = copyDir
+		}
+
```

Comment:
Using `if … copyFile` else … `copyDir` here would be simpler than adding the variable.

Reply:
done